services/coverage.js

```diff
@@ -16,21 +15,6 @@ function startKarma(overrides) {
 
     var serverHasStarted = false;
 
-    karmaConfig.files.forEach((x, i) => {
-        if (stringManipulation.startsWithSlash(x)) {
-            karmaConfig.files[i] = x.substring(1);
-        }
-    });
-
-    var processorKeys = Object.keys(karmaConfig.preprocessors);
-    processorKeys.forEach((key) => {
-        if (stringManipulation.startsWithSlash(key)) {
-            karmaConfig.preprocessors[key.substring(1)] = ["coverage"];
```

Comment:
Okay, I think this is okay, but I'm not sure why we make it `["coverage"]` every time before and didn't reuse whatever was actually _in_ `karmaConfig.preprocessors[key]`. That seems like an error in the earlier code (that's not important now that we aren't changing key names).